package.json

```
@@ -26,14 +26,14 @@
   "dependencies": {
     "async": "^2.6.3",
     "lodash": "^4.17.15",
-    "mongodb": "~2.2.31",
+    "mongodb": "^3.5.7",
     "mongodb-uri": "^0.9.7",
     "progress": "~1.1.8",
     "prompt": "~0.2.14",
     "semver": "^6.3.0"
   },
   "peerDependencies": {
-    "bedrock": "1.11.0 - 3.x"
+    "bedrock": "3.x"
```

I think this is equivalent, but for consistency with our other packages:

```suggestion
    "bedrock": "^3.0.0"
```

---

Might as well make it ^3.1.1 while we're at it.

---

@dlongley `3.x` is consistent with our other packages: https://github.com/digitalbazaar/bedrock-vc-issuer/blob/5b291c5b101f5bf892931387ab91a2f9d728d08c/package.json#L47-L63 but I will change it to `^3.1.1` shortly.

---

Version addressed here: https://github.com/digitalbazaar/bedrock-mongodb/pull/33/commits/4f74e384eac4ee82c5597f7cfe87ebcd982cabeb